### PR TITLE
Remove broken link.

### PR DESCRIPTION
**Pull request by eesheesh**

### CODE REVIEW

**Review by dem1k — APPROVED** [review]

**Review by kevinthecheung — APPROVED** [review]

**Review by Patrick333300 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -33,4 +33,3 @@ If you use TypeScript, you can install [`@types/gapi`](https://www.npmjs.com/pac
 ## Links
 
 - [Discussions](https://github.com/google/google-api-javascript-client/issues)
-- [Historical Announcements](http://google-api-javascript-client.blogspot.com/)
```
Comment (MattyPe) [inline README.md:36]: @mat.penicka@gmail.com

**Comment by aungmyolwina172-ship-it** [comment]
https://github.com/google/google-api-javascript-client.git

**Comment by aungmyolwina172-ship-it** [comment]
ATwin

**Comment by aungmyolwina172-ship-it** [comment]
fb.com/stars

**Comment by aungmyolwina172-ship-it** [comment]
fb.com/stars

**Pull request merged**